core/arch/arm/include/mm/tee_mm_def.h

```diff
@@ -32,37 +32,4 @@
 #define SMALL_PAGE_MASK		0x00000fff
 #define SMALL_PAGE_SIZE		0x00001000
 
-/*
```

Comment:
Is it possible to move `SMALL_PAGE_xxx` macros somewhere else to remove `tee_mm_def.h`?